.github/workflows/deflake-ui-tests-on-self-hosted.yml

```diff
@@ -18,11 +18,11 @@ defaults:
 jobs:
   build-then-test:
 
-    runs-on: [self-hosted, xcode11]
+    runs-on: macos-latest
```

Comment:
this workflow have to run on self hosted if we change label to `macos-latest` then it goes to github 

please use `[self-hosted, x86_64]`

Reply:
Unfortunately it didn't help.
`No runner matching the specified labels was found: self-hosted, x86_64.`
https://github.com/sparta-science/connect/actions/runs/725209184